lib/pure/json.nim

```diff
@@ -936,18 +935,6 @@ when defined(js):
       `result` = `x`.length;
     """
 
-  proc `[]`(x: JSObject, y: string): JSObject =
-    assert x.getVarType == JObject
```

Comment:
This assert is unnecessary.

Reply:
~~what if i call this proc with `12.toJs()`, wouldn't `x.getVarType != JObject`~~

is it because it's not exported and callers check for this ?

Reply:
The caller already make sure it is JObject or JArray in the below codes.